src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -176,7 +176,8 @@
 
     <ItemGroup>
       <FunctionalTest>
-        <Command>$(HelixPythonPath) $(RunnerScript) --dll %(Filename).dll $(OtherRunnerScriptArgs) -- $(XunitArgs)</Command>
+        <Command Condition="'$(TargetsWindows)' == 'true'">$(HelixPythonPath) $(RunnerScript) --script Runtests.cmd %HELIX_CORRELATION_PAYLOAD% %HELIX_WORKITEM_ROOT%\execution</Command>
```

Comment:
I prefer this mode.  In this case, the contents of the work item zip are in a folder under %HELIX_WORKITEM_PAYLOAD% (including RunTests.cmd/sh).  Doing it this way allows me to quickly debug in the case of a work item which both brings a DLL along (say, System.Collections.dll) and happens to have build a P2P reference to Foo.dll along as well, making the process non-destructive / repeatable.